src/aria2p/interface.py

```diff
@@ -577,6 +629,65 @@ def process_keyboard_event_main(self, event):
         elif event.key_code in Keys.UN_SELECT_ALL:
             pass  # TODO
 
+        elif event.key_code in Keys.MOVE_HOME:
+            if self.focused > 0:
+                self.focused = self.rows.index(self.rows[0])
+                logger.debug(f"Move focus home: {self.focused}")
+                if self.focused < self.row_offset:
+                    self.row_offset = self.focused
+                elif self.focused >= self.row_offset + (self.height - 1):
+                    self.row_offset = self.focused + 1 - (self.height - 1)
+                self.follow = None
+                self.refresh = True
+
+        elif event.key_code in Keys.MOVE_END:
+            if self.focused < len(self.rows) - 1:
+                self.focused = self.rows.index(self.rows[-1])
```

Comment:
Based on the description you give ("move to last download"), I understand it moves focus to the absolute bottom of the window, and not the currently displayed bottom row? Then you can simply do:

```suggestion
                self.focused = len(self.rows) - 1
```